internal/webhook/feature/features.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package feature
+
+import (
+	utilfeature "github.com/jetstack/cert-manager/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+)
```

Comment:
nitpick(non-blocking): I'm not sure that this import block follows our documented [coding conventions](https://cert-manager.io/docs/contributing/coding-conventions/).

That said, this check should (and will) be totally automated and I'm trying to bias towards pragmatism so it's certainly not a blocker and I'll LGTM without a change to this.

Reply:
Fixed!